composer.json

```diff
@@ -50,7 +50,8 @@
 		"npm-asset/fullcalendar": "^3.0.1",
 		"npm-asset/cropperjs": "1.2.2",
 		"npm-asset/imagesloaded": "4.1.4",
-		"monolog/monolog": "^1.24"
+		"monolog/monolog": "^1.24",
+		"psr/container": "^1.0"
```

Comment:
Please move the PHP dependencies before the `"fxp/composer-asset-plugin"` line and sort them alphabetically.